utils/validate-data-source.js

```diff
@@ -2,15 +2,20 @@ const appRoot = require('app-root-path');
 const config = require('config');
 const _ = require('lodash');
 
+const { openapi } = appRoot.require('utils/load-openapi');
+
 const { dataSources } = config.get('dataSources');
 const json = dataSources.includes('json')
-  ? appRoot.require('api/v1/db/json/fs-operations').validateJsonDb
+  ? appRoot.require(`/api${openapi.basePath}/db/json/fs-operations`).validateJsonDb
```

Comment:
When using `appRoot` require, I think we don't need the leading `/`. In addition, looks like the logic of these validation methods are pretty much the same, do you think we can write a generic function in the skeleton to share the similar logic?